### ISSUE TITLE
Unable to pry carpet

### ISSUE THREAD

**Issue by Ghagliiarghii** [space-wizards/space-station-14#30032]
## Description

A few days ago I was trying to make a vox box in the corner of a bar with a carpet down (Box station) but could not figure out how to pry up the carpet to layer pipes etc. underneath them to get that done. I'm not sure if this is a carpeted tile being immortal on Box (similar to grass on an expeditions) or a carpet in general issue. None of the tools in the engineering toolbelt worked to pry the tile (I didn't try RCD because there were people about.)

We should be able to pry up carpeted floors or remove carpet and then pry up the wood floor or whatever it was sitting on in order to get engineering work done.

**Reproduction**

Go to the bar in box station, attempt to pry the carpeted tile to view the underfloor.
Alternatively, place a carpeted floor down and attempt to pry it.

**Screenshots**

Here:
![image](https://github.com/user-attachments/assets/d4f7549d-c986-46f2-97c3-a0a053e1c800)

**Comment by JIPDawg** [comment]
I know for a fact you can use a knife on harm mode to pry it. You may be able to use wire cutters as well but I'm not sure.

**Comment by lzk228** [comment]
![image](https://github.com/user-attachments/assets/cb577298-2665-43d2-b07e-7633832486c8)

**Comment by luizwritescode** [comment]
TLDR: carpets are yet to become tiles

```
# TODO move all of this to tiles once tile smoothing is supported
- type: entity
  id: CarpetBase
  parent: BaseStructure
```
Colored carpets seem to be a workaround until tile smoothing is supported, so when they are on the floor they are not really tiles you can pry unlike the office carpet or clown carpet which are actually tiles. 

They are treated as structures that are also butcherable, so destroying one (just punch it) will drop the associated item, while using a knife will drop cloth. As far as I know you can't craft these colored carpets from cloth.

Using an RCD on them is not implemented, even though I'm pretty sure RCD has support to remove any structure.

Weirdly, if you are disassembling a fancy table, prying it with a crowbar WILL drop a carpet

The chapel carpet is even weirder as it does not have an associated item and will just cease to exist if you punch it, it's just there as a placeholder until tiles support rotating sprites

**Comment by IProduceWidgets** [comment]
I'm actually not sure if carpets as tiles is the way to go because then you have to worry about whats beneath them. I dunno.

**Comment by Velken** [comment]
Tile Stacking now exists, someone just needs to make carpet into tiles and fix the maps.
